docs/source/reference/_create_conda_env.rst

```diff
@@ -0,0 +1,383 @@
+Create bluesky environment
+==========================
+
+Describes the steps to create a virtual environment for a bluesky instrument
+(see `setup a bluesky instrument
+<https://bcda-aps.github.io/bluesky_training/instrument/_install_new_instrument.html#setup-a-bluesky-instrument>`__).
+
+A bluesky instrument uses many Python packages not included in the Python
+Standard Library. Rather than add these directly into the system Python
+installation, it is recommended to create a Python virtual environment which has
+the suite of packages (and specific versions) required.
+
+.. raw:: html
+
+    <details>
+    <summary>More about conda virtual environments</summary>
+
+    A virtual environment gives your control over the suite of installed
+    packages and insulates you from any system software updates. It is likely
+    that you will have more than one conda environment available.  Each of these
+    has a complete installation of Python with a suite of packages.  The suite
+    may differ between the environments, as well as the package versions. Even
+    the Python version itself may be different.
+
+    <ul>
+    <li>Each conda environment has a name, which corresponds to the file
+      directory in which the environments packages, libraries, and other
+      resources are stored. </li>
+    <li>When you install or update package(s) in one environment, this will not
+      affect any of the other environments. This makes it easy to control
+      which software is installed.</li>
+    </ul>
+
+    The <code>base</code> environment refers to the default environment created
```

Comment:
Could have used reST syntax instead of HTML:

HTML:

```
<code>base</code>
```

or reST:

```
``base``
```

Reply:
This is in a ` ..raw::html` section (details pull down), so rst syntax does not work:
![image](https://github.com/BCDA-APS/bluesky_training/assets/39918069/1f04df36-3832-4522-85e8-6c79fea32123)

Reply:
I'm blind.  Thanks.